integration-tests/volumes/client/scenarios_204.yml

```diff
@@ -1,6 +1,6 @@
 scenarios:
-  - name: success_204_No_Content_exluded_region
-    description: Test that Contile returns a 204 No Content for excluded regions
+  - name: success_200_No_Content_exluded_region
```

Comment:
This change seems unrelated to the changes to support Mobile.

Did we change anything about the `CONTILE_EXCLUDED_COUNTRIES_200` configuration option?

Reply:
Yes. I believe that this was covered by https://mozilla-hub.atlassian.net/browse/CONSVC-1274

Reply:
The `integration-tests/volumes/client/scenarios_204.yml` is used by `docker-compose.204.yml` which sets:

```yml
services:
  contile:
    environment:
      CONTILE_EXCLUDED_COUNTRIES_200: 0
```

Unless this has changed, we'd still expect a `204 No Content` with the configuration setting set to `0`.